<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoWindow</class>
 <widget class="QWidget" name="VideoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border:1px rgb(85, 255, 0);
</string>
  </property>
  <widget class="QLabel" name="label_videoLoss">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>283</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/videoloss.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_motionDetection">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/motetion_warn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_setHome">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>360</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">

 QPushButton {
      
		border-image: url(:/images/setHome.png);

  }

  QPushButton:pressed {
		border-image: url(:/images/setHome_p.png);

  }
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_video_swith">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>360</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">
 QPushButton {
background-color: transparent;
		border-image: url(:/images/videoType_swith.png);
  }

  QPushButton:pressed {
		border-image: url(:/images/videoType_swith_p.png);

  }
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_replay">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>360</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">
 QPushButton {
      
		border-image: url(:/images/video_replay.png);

  }

  QPushButton:pressed {
		border-image: url(:/images/video_replay_p.png);

  }
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_cloudControl" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>160</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-image: url(:/images/cloudControl_bg.png);</string>
   </property>
   <widget class="QPushButton" name="pushButton_cloudControl_up_3">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>10</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
 QPushButton {
      
		border-image: url(:/images/cloudControl_up.png);

  }

  QPushButton:pressed {
		border-image: url(:/images/cloudControl_up_p.png);

  }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cloudControl_down">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>138</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
 QPushButton {
		border-image: url(:/images/cloudControl_down.png);
  }

  QPushButton:pressed {
		border-image: url(:/images/cloudControl_down_p.png);
  }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cloudControl_left">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
 QPushButton {
		border-image: url(:/images/cloudControl_left.png);
  }

  QPushButton:pressed {
		border-image: url(:/images/cloudControl_left_P.png);
  }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cloudControl_right">
    <property name="geometry">
     <rect>
      <x>134</x>
      <y>70</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
 QPushButton {
		
		border-image: url(:/images/cloudControl_right.png);
  }

  QPushButton:pressed {
		border-image: url(:/images/cloudControl_right_p.png);
  }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_ipc_identify">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>92</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font-family: &quot;Microsoft YaHei&quot;;
	font-size:14px;
    color: #ffffff;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_recordState">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>57</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/record_state.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
